source/reference/mongodb-metrics.csv

```diff
@@ -0,0 +1,6 @@
+Metric Name,Metric Key,Unit Type,Relation,Relation Units Type
```

Comment:
Metrics for each DB are stored as a CSV file and imported for ease of maintenance. The info comes from confluence and is incomplete there.